test/unit/main/lib/icon/storage/IconFileStorage.test.js

```diff
@@ -0,0 +1,127 @@
+import test from 'ava';
+import path from 'path';
+import pify from 'pify';
+import fsLib from 'fs';
+import tmpLib from 'tmp';
+import IconFileStorage from '../../../../../../src/main/lib/icon/storage/IconFileStorage';
```

Comment:
Lol.. @sallar I wonder if there'd be any benefit from bundling into a single file for the app.. maybe before testing. Not sure what we currently do here.. but we do bundle at least the front-end right?

Reply:
lol I have no idea what is going on here! :D @perry-mitchell @hugo-cardenas

Reply:
Well, I just went with replicating source directory structure for tests... which unfortunately leads to this.
Alternatives to this ugliness:
- Define a different test directory structure
- Place tests together with code
- Use some utility or config to allow absolute imports from root, so that this would be something like the following, no matter where you call it from:
```javascript
import foo from 'src/main/lib/icon/storage/IconFileStorage'
```

@perry-mitchell Not sure I understand what you mean in your comment.

Reply:
No.. was just laughing inwardly at how deep the structure is already. I don't think there are any action points here at this stage (at least not in this PR), but maybe something we can look at in the future..